### PR TITLE
fix #76254 autohero.com

### LINKED ISSUE #76254: autohero.com
### Issue URL (Annoyance)
 [https://www.autohero.com/de/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.autohero.com%2Fde%2F)
### Comment
Username: `@Morku`
> Cookie Banner
> --
> Die folgenden Informationen werden automatisch generiert. Bitte löschen Sie sie nicht.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.user.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Firefox
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br>Send Do-Not-Track header, <br>Remove X-Client-Data header from HTTP requests, <br>Strip URLs from tracking parameters
DNS filtering: | server: `Cloudflare DNS-over-HTTPS`<br>filters: `"AdGuard DNS filter" with AdGuard Home`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Russian, <br/>AdGuard German

### PR DESCRIPTION
**Pull request by wout-codes**
#76254

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -92,6 +92,7 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 rshb.ru###alert.bottom
+autohero.com##header > div > [class^="root"]
```
Comment (zloyden) [inline AnnoyancesFilter/sections/cookies_specific.txt:95]: Something like that `autohero.com##header > div[class^="container_"] > div[class^="body_"] + div[class^="root_"]` will be more accurate.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:95]: Okay. I will check again.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt:95]: @zloyden How to use `~` and `+`?

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: There is a useful [information ](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors)about it.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @zloyden I still don't understand :( it doesn't have pictures/diagrams to explain properly.

Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Just imagine a tree. You choose a branch with 10 leaves in it. Let's say you need to take only the third sheet. Select the previous element and specify the third one, this is a kind of additional condition. The result is 2+3 `div[class^="body_"] + div[class^="root_"]`. But if this doesn't work, you can change the condition to a more previous element, but the result will be 1~3 `div.someclassname ~ div[class^="root_"]`.

Comment (wout-codes) [reply AnnoyancesFilter/sections/cookies_specific.txt]: Thanks @zloyden. Actually I am a Software Engineer, who is on vacation, so I don't really focus on this (CSS Selectors). I am not disagreeing that I didn't learn it, as I learned it when I was doing my course in University. Mainly I focus on C++, so I might forget all these.

**Pull request merged**